test/studies/prk/stencil/stencil-pretty.chpl

```diff
@@ -198,6 +201,7 @@ proc main() {
     if useStencilDist then {
       if debug then diagnostics('output.updateFluff()');
       output.updateFluff();
```

Comment:
(Same for other file): I don't think we need to update the output, since we're just writing to it. That might explain some of the performance difference on 16-node-xc.